debian/base/Dockerfile.rpm

```diff
@@ -0,0 +1,94 @@
+#
+# Copyright 2016 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+FROM centos:centos7
+
+ARG COMMIT_ID=unknown
+LABEL io.confluent.docker.git.id=$COMMIT_ID
+ARG BUILD_NUMBER=-1
+LABEL io.confluent.docker.build.number=$BUILD_NUMBER
+
+ARG CONFLUENT_RPM_REPO="http://packages.confluent.io"
+# TODO: see about enabling this for rpms
+# ARG APT_ALLOW_UNAUTHENTICATED=false
+# #Set an env var so that it's available in derived images
+# ENV APT_ALLOW_UNAUTHENTICATED=${APT_ALLOW_UNAUTHENTICATED}
+
+MAINTAINER partner-support@confluent.io
+LABEL io.confluent.docker=true
+
+# # Python
+# # TODO: get this version
+# # ENV PYTHON_VERSION="2.7.9-1"
```

Comment:
What do we need Python 2 for? We should remove this comment / avoid using Python 2 unless there is a hard reason that we can't use Python 3.